apps/web-client/src/lib/db/cr-sqlite/note.ts

```diff
@@ -56,6 +118,18 @@ export function createOutboundNote(db: DB, noteId: number): Promise<void> {
 	});
 }
 
+/**
+ * Retrieves all uncommitted inbound notes with their associated warehouse and book totals.
+ * Only returns notes that have not been committed (draft state).
+ *
+ * @param {DB} db - Database connection
+ * @returns {Promise<InboundNoteListItem[]>} Array of inbound notes with:
+ *   - id: Note identifier
+ *   - displayName: Human readable name
+ *   - warehouseName: Associated warehouse name
+ *   - updatedAt: Last modification timestamp
+ *   - totalBooks: Sum of all book quantities in note
+ */
 export async function getAllInboundNotes(db: DB): Promise<InboundNoteListItem[]> {
```

Comment:
Not the focus of this PR, but I strongly believe this function should have "uncommitted" in its name somehow.

Reply:
`getUncommittedInboundNotes` is a little unreadable. `getAtiveInboundNotes`?